healthcheck/generalchecks/bnd.bnd

```diff
@@ -1,15 +1,11 @@
-Bundle-Category: healthcheck
-
-Bundle-Description: ${project.description}
-
-Bundle-DocURL: https://felix.apache.org
-
-Bundle-License: Apache License, Version 2.0
-
-Bundle-Vendor: The Apache Software Foundation
-
-Conditional-Package: org.apache.commons.cli.*,org.apache.felix.utils.*
-
-Export-Package: org.apache.felix.hc.generalchecks.util
-
-Import-Package: org.apache.felix.rootcause*;resolution:="optional", *
\ No newline at end of file
+Bundle-Category     : healthcheck
+Bundle-Description  : ${project.description}
+Bundle-DocURL       : https://felix.apache.org
+Bundle-License      : Apache License, Version 2.0
+Bundle-Vendor       : The Apache Software Foundation
+Export-Package      : org.apache.felix.hc.generalchecks.util
+Import-Package      : org.apache.felix.rootcause*;resolution:="optional", *
+-conditionalpackage : \
+	org.apache.felix.utils.*,\
+	org.apache.commons.cli.*,\
+	org.apache.commons.lang3.*
```

Comment:
@amitjoy If you use this bundle, embedding org.apache.commons.lang3 in both core and here could in total already result in a larger memory footprint (obviously depends on how much you actually use of the bundle)

Reply:
Yeah, I agree completely with you. This results in extra ~250KB in the respective bundles. I will think about some approaches that might help us trim down the file sizes.